config_management/group_vars/all

```diff
@@ -2,7 +2,7 @@
 go_version: 1.7.4
 terraform_version: 0.8.5
 docker_version: 1.11.2
-kubernetes_version: 1.5.2
+kubernetes_version: 1.6.0-beta.2
```

Comment:
Agreed, let me try the patch I listed here: https://github.com/weaveworks/build-tools/pull/81#discussion_r105685564